app/src/interfaces/_system/system-fields/system-fields.vue

```diff
@@ -100,6 +107,15 @@ const fields = computed<(Field & { key: string })[]>({
 
 const { t } = useI18n();
 
+function selectAll() {
+	const newArray = fieldsStore.getFieldsForCollection(props.collectionName).map((field) => field.field);
+	emit('input', newArray);
+}
+
+function deselectAll() {
+	emit('input', null);
+}
```

Comment:
I'm wondering if this is required in this scope. You can also already click the field label and remove the value there. You now click "Add Field" to then "Remove" all fields, which feels a little counter-intuitive